HLTrigger/Configuration/python/HLT_75e33/modules/hltInitialStepTrajectorySeedsLST_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+hltInitialStepTrajectorySeedsLST = cms.EDProducer('LSTOutputConverter',
+    lstOutput = cms.InputTag('hltLST'),
+    phase2OTHits = cms.InputTag('hltPhase2OTHitsInputLST'),
+    lstPixelSeeds = cms.InputTag('hltPixelSeedInputLST'),
+    includeT5s = cms.bool(True),
+    includeNonpLSTSs = cms.bool(True),
+    propagatorAlong = cms.ESInputTag('', 'PropagatorWithMaterial'),
+    propagatorOpposite = cms.ESInputTag('', 'PropagatorWithMaterialOpposite'),
+    SeedCreatorPSet = cms.PSet(
+        ComponentName = cms.string('SeedFromConsecutiveHitsCreator'),
+        propagator = cms.string('PropagatorWithMaterial'),
+        SeedMomentumForBOFF = cms.double(5),
+        OriginTransverseErrorMultiplier = cms.double(1),
+        MinOneOverPtError = cms.double(1),
+        magneticField = cms.string(''),
+        TTRHBuilder = cms.string('WithTrackAngle'),
```

Comment:
just a curiosity. 
Can one use here `hltESPTTRHBuilderWithTrackAngle` instead? 
I think we have a proliferation of `ESProducer`s doing the same thing in the menu.

Reply:
Yes, should be possible (my oversight not using the proper HLT one). Let me do it in one go after the presentation, together with any potential feedback I get there.

Reply:
This should be fixed in 1281d72.